src/nidcpower/disabled_examples/nidcpower_advanced_sequence.py

```diff
@@ -26,7 +26,7 @@ def example(resource_name, channels, options, steps, voltage_start, voltage_fina
     with nidcpower.Session(resource_name=resource_name, channels=channels, options=options) as session:
 
         session.source_mode = nidcpower.SourceMode.SEQUENCE
-        session.source_delay = datetime.timedelta(seconds=0.1)
+        session.source_delay = hightime.timedelta(seconds=0.1)
```

Comment:
I see you changed client code. I think that's fine.
But do we have system tests in which we exercise and verify the higher precision of hightime?

Reply:
Given that system tests run on simulated sessions, it's difficult to verify the higher precision of `hightime` in system tests. I've updated nifake unit tests to verify the precision, in both properties and method parameters.